packages/datadog-plugin-fs/test/index.spec.js

```diff
@@ -1329,16 +1330,28 @@ describe('Plugin', () => {
       })
 
       if (realFS.Dir) {
-        describe.skip('Dir', () => {
+        describe('Dir', () => {
           let dirname
           let dir
+          function waitToEmptyActiveRequests (pendingIterations) {
```

Comment:
I can see two possible easy fixes:
1) Add this wait in [packages/dd-trace/test/setup/core.js](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/test/setup/core.js) file and all the test will wait until activeRequests === 0
2) Configure the maxActiveRequests to something like 1 000 for the tests, and request.writable will be always true

What do you think?

Reply:
Thinking more about this I'm actually realizing that this should never happen in the first place because the request needs to be done by the time the next test runs, so it's unclear why there would ever be more than 1 request to the agent at any given time. I think we should make sure to understand the problem first before looking into possible solutions.

Reply:
You are right, call to this `waitToEmptyActiveRequests` in `core.js` wouldn't fix anything.

Basically, we are creating and finishing a lot of spans in these test. For example, in Dir -> Symbol.asyncIterator test, we are creating and finishing a parent span in one `beforeEach`, creating one dir and 3 files, opening the dir and iterating the dir (directory with 3 items, it means 4 times `iterator.next()` and other span to close the dir. A total of 10 spans (+ the spans created in afterEach)
If we have the limitation of 8 spans, these flaky test will exist